### ISSUE TITLE
Functions in pcg32.v should return option, rather than forcing exit

### ISSUE THREAD

**Issue by chipnetics** [vlang/v#13556]
**V version:** 
V 0.2.4 da9331f.05e946e

**OS:** 
linux, Debian GNU/Linux 11 (bullseye)

**What did you do?**
```
import rand

fn main()
{
	random := rand.f32_in_range(4.12,3.25)
}
```

**What did you expect to see?**
Compiler error that specifies function returns an option that must be handled or propogated.
 
**What did you see instead?**
`max must be greater than min` followed by forced program exit.

User should be able to handle situations where max less than min, rather than V taking full control and forcing program exit.  The eprintln messages may also then be removed.
See source: https://github.com/vlang/v/blob/master/vlib/rand/pcg32/pcg32.v

**Comment by dumblob** [comment]
That'd slow down the functions a lot with no way of recovering the current speed. So I think this is a perfect example of writing your own downstream wrappers.

**Comment by hungrybluedev** [comment]
Fixed in #13570.
